src/coreclr/src/jit/optcse.cpp

```diff
@@ -2159,13 +2159,13 @@ class CSE_Heuristic
 #else                                 // _TARGET_ARM_
                     if (hugeFrame)
                     {
-                        cse_def_cost = 10 + (2 * slotCount); // movw/movt r10 and str reg,[sp+r10]
-                        cse_use_cost = 10 + (2 * slotCount);
+                        cse_def_cost = 10 + 2; // movw/movt r10 and str reg,[sp+r10]
```

Comment:
there is a semantic change here, so it is not only `refactor `, are you aware of that? 
Do you want to multiply 10 by `slotCount`? 
Where do these consts come from? 
Replace `10 + 2` with `12`?

Reply:
Yes I am aware of this semantic change.  It is actually correct, as the cost to generate load when we have a huge frame on ARM32 in terms of code size is 12 bytes for each slot.

For SMALL_CODE we measure the costs in terms of how many bytes of code will be generated.

It look slike I need to visit the code that set hugeFrame to ensure that it does not get set for ARM64.